packages/contentstack-import/src/import/modules/locales.ts

```diff
@@ -137,8 +137,12 @@ export default class ImportLocales extends BaseClass {
       fsUtil.writeFile(this.langUidMapperPath, this.langUidMapper);
     };
     const onReject = ({ error, apiData: { uid, code } = undefined }: any) => {
-      log(this.importConfig, `Language '${code}' failed to import`, 'error');
-      log(this.importConfig, formatError(error), 'error');
+      if (error?.errorCode === 247) {
+        log(this.importConfig, formatError(error), 'info');
+      } else {
+        log(this.importConfig, `Language '${code}' failed to import`, 'error');
+        log(this.importConfig, formatError(error), 'error');
```

Comment:
@shafeeqd959 Even this line looks like same in both the if and else blocks except the print type (info/error). BTW, Why is the error printed as info.?